arcaflow_plugin_coremark_pro/coremark_pro_plugin.py

```diff
@@ -16,10 +16,13 @@
 )
 
 
-def run_oneshot_cmd(command_list, workdir) -> str:
+def run_oneshot_cmd(command, workdir) -> str:
     try:
         cmd_out = subprocess.check_output(
-            command_list, stderr=subprocess.STDOUT, text=True, cwd=workdir
+            command,
+            stderr=subprocess.STDOUT,
+            text=True,
+            cwd=workdir,
         )
```

Comment:
You don't want to revert this change?

Reply:
It just doesn't matter much.

Reply:
This change has zero impact on the code's execution but it will show up in a future retrospection or forensic search, and it would be best to avoid that sort of "false positive" (and, if you don't commit this change, then it cannot be the basis for a "positive" which is _not_ "false").